moto/networkmanager/responses.py

```diff
@@ -109,3 +109,150 @@ def describe_global_networks(self) -> str:
         return json.dumps(
             dict(GlobalNetworks=list_global_networks, nextToken=next_token)
         )
+
+    def create_site(self) -> str:
+        params = json.loads(self.body)
+        global_network_id = unquote(self.path.split("/")[-2])
+        description = params.get("Description")
+        location = params.get("Location")
+        tags = params.get("Tags")
+        site = self.networkmanager_backend.create_site(
+            global_network_id=global_network_id,
+            description=description,
+            location=location,
+            tags=tags,
+        )
+        return json.dumps(dict(Site=site.to_dict()))
+
+    def delete_site(self) -> str:
+        global_network_id = unquote(self.path.split("/")[-3])
+        site_id = unquote(self.path.split("/")[-1])
+        site = self.networkmanager_backend.delete_site(
+            global_network_id=global_network_id,
+            site_id=site_id,
+        )
+        return json.dumps(dict(Site=site.to_dict()))
+
+    def get_sites(self) -> str:
+        params = self._get_params()
+        global_network_id = unquote(self.path.split("/")[-2])
+        site_ids = params.get("siteIds")
```

Comment:
Looks like the `_get_params()` -method isn't quite smart enough to understand this usecase @zkarpinski.

You can directly get these values from the querystring, that will show all values:
```
self.querystring.get("siteIds")
```

Reply:
Thanks @bblommers ! I made the corrections.